docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '2'
+
+services:
+
+  gtsite-service:
+    image: fosskers/gtsite-service
```

Comment:
The previous config (just for the site, no demos) did something similar to what I'm doing here, except it tagged the images as `quay.io/geotrellis/foobar`.